DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -481,3 +483,41 @@ def run_protocol(self, device, command, *argv, **kwarg):
 
     def parse_protocol(self, command, output, *argv, **kwarg):
         return command
+
+    def format_send_ping(self, command, *argv, **kwarg):
+        return command
+
+    def run_send_ping(self, device, command, *argv, **kwarg):
```

Comment:
I propose to make "send_ping" and "send_arp" two separate functions (APIs) since in the future, if we need to modify any functionality of send_ping or send_arp, this function will become more complicated.

Reply:
Splitting them would mean introducing duplicate code. Internally they are using the same logic to find the IP interface and operate on it.

Reply:
You may move common code to some function like _get_ip_interface and then use it for both send_ping and send_arp functions.

Reply:
already working on it 🙂